tensorrt_llm/_torch/speculative/mtp.py

```diff
@@ -852,7 +852,7 @@ def sample_and_accept_draft_tokens(
 
         # Strict acceptance
         else:
-            if self.is_thop:
+            if False:
```

Comment:
What is happening here?

Reply:
We did not implement the C++ MTP sampling kernel for thop, so this is a hack to use the pytorch implementation of the sampler. I'll revert this line for now, but I'm not sure if the code added by this PR will work properly without forcing this case.